static/js/cluster-stats.js

```diff
@@ -72,14 +72,12 @@ function renderChart() {
         dataType: 'json',
         data: JSON.stringify(data)
     })
-        .then((res)=> {
-            $('#app-content-area').show();
-            drawStatsChart(res,data)
-        })
-        .catch(showCStatsError);
+    .then(res => drawStatsChart(res, data, "logs").then(() => drawStatsChart(res, data, "metrics")))
+    .catch(showCStatsError);
```

Comment:
Why do you no longer need `$('#app-content-area').show();`? (I don't know what this does)

Reply:
`$('#app-content-area').show();` is used to show the cluster stats div, and it was already being shown by default. I'm not very familiar with the UI code and I don’t know if there might be a case where this is required, so I added it back.